docs/user-guide/nodes.rst

```diff
@@ -401,13 +401,15 @@ On the Pulp server to be used as the parent node:
   $ sudo yum install pulp-nodes-parent pulp-nodes-admin-extensions
   $ sudo service httpd restart
```

Comment:
Sorry, just noticed the example includes restarting http. The http restart needs to happen after the oauth configuration change. The example needs to be split. For consistency with On The Child instructions, you probably should cut-n-paste from there.